### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by sammy-ette** [sammy-ette/Hilbish#35] [Ideas]
Every cool software has some kind of logo and I definitely thing Hilbish should have one as well
no ideas though for it

**Comment by marauder316** [comment]
Alright, @TorchedSammy, I got a couple. Here are the [Looka](https://looka.com/) links and their screenshots:

**[No. 1 - Brushlike Font and Hibiscus Flower](https://looka.com/s/64241269)**
![image](https://user-images.githubusercontent.com/72051451/114105860-d1efe200-988a-11eb-9c69-e8fbb14cbb16.png)

**[No. 2 - Sans Serif Font and Terminal Icon](https://looka.com/s/64241295)**
![image](https://user-images.githubusercontent.com/72051451/114105972-0bc0e880-988b-11eb-8173-6ff8ba51e998.png)


**[No. 3 - Cursive "H" in a Brushed Box with Small Monospace Font](https://looka.com/s/64241334)**
![image](https://user-images.githubusercontent.com/72051451/114106135-593d5580-988b-11eb-9a14-3d906ca475bc.png)

**Reply by sammy-ette** [reply]
if i had to pick 1 specific one, the 1st is the best option though the font should be something different
having the hibiscus is good there

**Reply by marauder316** [reply]
Cool, what would you want the font to look like?

I could also grab white/black and black/white monochromes.

**Reply by sammy-ette** [reply]
SF Pro Display yo

**Comment by sammy-ette** [comment]
so with some of the proposals here, ill get to working on it myself

**Reply by marauder316** [reply]
👍🏾

**Comment by sammy-ette** [comment]
logo has been made (like a week ago)
